Source/Core/Core/MemoryWatcher.h

```diff
@@ -37,8 +41,13 @@ class MemoryWatcher final
 
   bool m_running;
 
+#ifdef _WIN32
+  void* m_pipe;
+  unsigned long m_written;
```

Comment:
On a second look, this looks unnecessary to expose, you're only using it to pass a non-nullptr value to WriteFile, which you never check or otherwise use. If there's no interest in keeping this information around, just declare it as a dummy above the WriteFile call.

Reply:
Fixed.